### PR TITLE
616 integration event function healtcheck

### PR DESCRIPTION
**Pull request by kristianschneider**
This implements healthcheck for live and 
SQL Db
Meteringpoint created servicebus topic
Meteringpoint created servicebus subscription
Meteringpoint connected servicebus topic
Meteringpoint connected servicebus subscription

### CODE REVIEW

File: source/IntegrationEventListener/source/Energinet.DataHub.Aggregations.IntegrationEventListener/Common/HealthCheckConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        public static void AddHealthChecks(this IServiceCollection services, HostBuilderContext context)
+        {
+            //Health checks
+            services.AddScoped<IHealthCheckEndpointHandler, HealthCheckEndpointHandler>();
+            services.AddHealthChecks()
+                .AddLiveCheck()
+                .AddSqlServer(
+                    name: "MeteringPointMasterDataDb",
+                    connectionString: context.Configuration["DATABASE_MASTERDATA_CONNECTIONSTRING"])
```
Comment (BjarkeMeier) [inline source/IntegrationEventListener/source/Energinet.DataHub.Aggregations.IntegrationEventListener/Common/HealthCheckConfiguration.cs:33]: We should use `EnvironmentSettingNames.cs` like in the other function app for all these hardcoded strings tightly bound to infra.

Comment (kristianschneider) [reply source/IntegrationEventListener/source/Energinet.DataHub.Aggregations.IntegrationEventListener/Common/HealthCheckConfiguration.cs:33]: Ill pull in from the source branch

**Comment by ZavezX** [comment]
@kristianschneider somehow you have comitted some secrets, that we should take care of and have removed from the source code. Maybe we should consider using squash commits for our PR's?

**Comment by BjarkeMeier** [comment]
> @kristianschneider somehow you have comitted some secrets, that we should take care of and have removed from the source code. Maybe we should consider using squash commits for our PR's?

I'm not sure that squash is enough. The secrets may already have been picked up and they are also available as long as the branch lives. I think you should reach out to MD regarding this exposure.

Regarding squash I'd love us to use squash as we as a team doesn't seem to have good practices regarding creating a helpful git history. I believe squashing will help improve this to some degree.

File: build/primary/main/func-integration-event-listener.tf [hunk 2]
```diff
@@ -23,17 +23,21 @@ module "func_integration_event_listener" {
...
     FUNCTIONS_WORKER_RUNTIME                              = "dotnet-isolated"
     INTEGRATION_EVENT_LISTENER_CONNECTION_STRING          = data.azurerm_key_vault_secret.sb_domain_relay_listener_connection_string.value
+    INTEGRATION_EVENT_MANAGER_CONNECTION_STRING            = data.azurerm_key_vault_secret.sb_domain_relay_manage_connection_string.value
     EVENT_HUB_CONNECTION                                  = module.evh_aggregations.primary_connection_strings["send"]
     EVENT_HUB_NAME                                        = module.evh_aggregations.name
-    CONSUMPTION_METERING_POINT_CREATED_TOPIC_NAME         = data.azurerm_key_vault_secret.sbt_consumption_metering_point_created_name.value
-    CONSUMPTION_METERING_POINT_CREATED_SUBSCRIPTION_NAME  = data.azurerm_key_vault_secret.sbs_consumption_metering_point_created_to_aggregations_name.value
+    METERING_POINT_CREATED_TOPIC_NAME         = data.azurerm_key_vault_secret.sbt_consumption_metering_point_created_name.value
+    METERING_POINT_CREATED_SUBSCRIPTION_NAME  = data.azurerm_key_vault_secret.sbs_consumption_metering_point_created_to_aggregations_name.value
```
Comment (BjarkeMeier) [inline build/primary/main/func-integration-event-listener.tf:40]: nit: indentation

**Review by BjarkeMeier — APPROVED** [review]

**Review by johevemi — APPROVED** [review]

**Pull request merged**